benchmarks/benchmark_ml_kem.py

```diff
@@ -53,14 +53,14 @@ def benchmark_ml_kem(ML_KEM, name, count):
     avg_dec = sum(dec_times) / count
     print(
         f" {name:11} |"
-        f"{avg_keygen*1000:8.2f}ms  {1/avg_keygen:11.2f}"
-        f"{avg_enc*1000:8.2f}ms  {1/avg_enc:10.2f}"
-        f"{avg_dec*1000:8.2f}ms  {1/avg_dec:8.2f}"
+        f"{avg_keygen*1000:7.2f}ms | {1/avg_keygen:10.2f} |"
+        f"{avg_enc*1000:6.2f}ms | {1/avg_enc:9.2f} |"
+        f"{avg_dec*1000:6.2f}ms | {1/avg_dec:7.2f} |"
     )
 
 
 if __name__ == "__main__":
-    count = 1000
+    count = 10
```

Comment:
that will cause a lot of variability in reported results...

Reply:
thanks for spotting this, I lowered it when i was fixing the formatting of the table and forgot to increase it.